remarkable_sync.py

```
@@ -0,0 +1,364 @@
+        sync = RemarkableSync(config)
+        
+        # Perform sync
+        uploaded, downloaded = sync.full_sync()
+        
+        print(f"\\n✅ Sync complete!")
+        print(f"📤 Uploaded {uploaded} files to reMarkable")
+        print(f"📥 Downloaded {downloaded} annotated files")
+        
+    except Exception as e:
+        print(f"❌ Sync failed: {e}")
+        print(f"💡 Make sure rmapi is installed and configured")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)